002-db-schema.md

```diff
@@ -0,0 +1,88 @@
+# 002 - Template
+
+DB Schema by use cases from [001-usecases.md](./usecases.md)
+
+## Status
+
+* 2018-04-27: proposed
+* 2018-05-09: updated
+
+## Context
+
+By the defined use cases in the previous ADR to design a DB schema and describe all data
+
+## Decision
+
+* Use RDMS like Postgres
+* Initial SQL script like the following: 
+
+```sql
+CREATE TABLE person (
+  id SERIAL PRIMARY KEY,
+  login VARCHAR(20) NOT NULL,
+  name VARCHAR(35) NOT NULL,
+  email VARCHAR(35) UNIQUE NOT NULL,
+  location VARCHAR(50),
+  bio VARCHAR,
+  avatar_url VARCHAR
+);
+
+CREATE TABLE organization (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(35) NOT NULL,
+  email VARCHAR(35) UNIQUE NOT NULL,
+  description VARCHAR,
+  location VARCHAR(50),
+  avatar_url VARCHAR
+);
+
+CREATE TABLE membership (
+  organization_id INT REFERENCES organization(id),
+  person_id INT REFERENCES person(id),
+  role VARCHAR CHECK (role IN ('owner', 'member')),
+  CONSTRAINT membership_id PRIMARY KEY (organization_id, person_id)
+);
+
+CREATE TABLE vacancy (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(30) NOT NULL,
+  description VARCHAR NOT NULL,
+  location VARCHAR(50),
```

Comment:
location can be a list of available location

Reply:
it can be, but for now it will be simple text, that user type by hands